src/patternfly/components/Popover/popover.scss

```diff
@@ -94,11 +129,78 @@
       transform: translateX(var(--pf-c-popover__arrow--m-right--TranslateX)) translateY(var(--pf-c-popover__arrow--m-right--TranslateY)) rotate(var(--pf-c-popover__arrow--m-right--Rotate));
     }
   }
+
+  &.pf-m-icon {
+    display: flex;
+  }
+
+  &.pf-m-default {
+    --pf-c-popover__title-icon--Color: var(--pf-c-popover--m-default__title-icon--Color);
+  }
+
+  &.pf-m-danger {
+    --pf-c-popover__title-icon--Color: var(--pf-c-popover--m-danger__title-icon--Color);
+  }
+
+  &.pf-m-warning {
+    --pf-c-popover__title-icon--Color: var(--pf-c-popover--m-warning__title-icon--Color);
+  }
+
+  &.pf-m-success {
+    --pf-c-popover__title-icon--Color: var(--pf-c-popover--m-success__title-icon--Color);
+  }
+
+  &.pf-m-info {
+    --pf-c-popover__title-icon--Color: var(--pf-c-popover--m-info__title-icon--Color);
+  }
+}
+
+.pf-c-popover__title,
+.pf-c-popover__title-text {
+  @include pf-text-overflow;
+}
+
+.pf-c-popover__title {
```

Comment:
I created an issue to add a `__title` and `__header` element to the popover to match other components (like the modal) here - https://github.com/patternfly/patternfly/issues/4123

I don't think we need the `__title` now, as we'll need to maintain 2 separate elements - this new `__title` and the existing `.pf-c-title` to avoid a breaking change. I think we can do everything we need for these alert variations working within the existing title.

Reply:
sure @mcoker , I have now removed all occurrences of .pf-c-popover__title from popover.scss